### PR TITLE
Added requirement plan-for-teams-live-events.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4221

@LanaChin please double check it. I'm almost sure about it because if you don't have schedule meetings available in Teams...

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ The user must have:
 - Screen sharing enabled in Teams meetings (*The TeamsMeetingPolicy -ScreenSharingMode parameter = EntireScreen*).
 - Live event scheduling in Teams enabled (*The TeamsMeetingBroadcastPolicy -AllowBroadcastScheduling parameter = True*).
 - Permissions to create live events in Stream (for external app or device production).
+- Coexistence mode configured to be able to schedule Teams meetings (*Islands, Meeting First or Teams only*).
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/plan-for-teams-live-events.md]:
```suggestion
- Coexistence mode configured to be able to schedule Teams meetings (*Islands, Meeting First, or Teams only*).
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Pinged PM to confirm

**Comment by LanaChin** [comment]
Thanks @dariomws and JohanFreelancer9, updates merged.

**Pull request merged**